### PR TITLE
Rep Considerations

### PR DESCRIPTION
**Pull request by KathrynN**
I know that you can either take Aldor or Scryer pumpkins, but i thought it might help if, were you hated with one, you weren't directed there in the first place :)

Only tested on my machine so would appreciate further testing

### CODE REVIEW

**Comment by EthanCentaurai** [comment]
I like this idea. I'll have a play with it when I get home from work.

**Comment by EthanCentaurai** [comment]
Below Neutral might be the best approach. I have an alt who is Neutral with both but hasn't done the quest to choose a side yet and she can access both buckets. (It was an interesting day as I was the only one there today. Quiet but productive!)

File: Alliance.lua [hunk 1]
```diff
@@ -223,6 +228,17 @@ points[111] = { -- "ShattrathCity"
 	[56208180] = 12404, -- Scryer's Tier
 }
 
+if standingWithAldor < neutralStanding then
+	points[104][61002820] = nil -- ShadowmoonValley Altar of Sha'tar
+	points[111][28104900] = nil -- ShattrathCity Aldor Rise
+	
+end
+
+if standingWithScryers < neutralStanding then
```
Comment (KathrynN) [inline Alliance.lua]: this doesn't working currently :\

Comment (KathrynN) [reply Alliance.lua]: Annoyingly, it works when you reload...  The GetFactionInfo doesn't appear to give correct info on initial addon load.

Comment (EthanCentaurai) [reply Alliance.lua]: I'll put a delay on it when I merge it in. That should do the trick.

Comment (EthanCentaurai) [reply Alliance.lua]: Wait I found it. The function is GetFactionInfoByID, not GetFactionInfo.

**Pull request merged**